CHANGELOG.md

```diff
@@ -16,6 +16,19 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ---
 
+## [2.2.9-bl-parametrize-max-rho] - 2024-10-29
+### Added
+- Add `renv.lock` file containing R packages and their version
+- Add `battenberg_bl_custom.R`, a customized `battenberg.R` script
+- Add `battenberg_wgs_bl_custom.R`, a customized `battenberg_wgs.R` wrapper
+
+### Changed
+- Reproduce R packages environment using `renv.lock` file
```

Comment:
@yashpatel6 and I discussed this briefly and since this docker repo is the only one that required granular version requirements, we may not require this for other repos at the moment.